lib/ansible/modules/cloud/scaleway/scaleway_sshkey.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python
+#
+# Scaleway SSH keys management module
+#
+# Copyright (C) 2018 Online SAS.
+# https://www.scaleway.com
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: scaleway_sshkey
+short_description: Scaleway SSH keys management module
+version_added: "2.6"
+author: Remy Leone (@sieben)
+description:
+    - "This module manages SSH keys on Scaleway account."
+
+options:
+  state:
+    description:
+     - Indicate desired state of the SSH key.
+    default: present
+    choices: ['present', 'absent']
+  ssh_pub_key:
+    description:
+     - The Public SSH key to add.
```

Comment:
Is that a path or a string ?

Reply:
It's a string.

Reply:
This could be added in the description, something like `as a string` (same as [authorized_key](http://docs.ansible.com/ansible/2.5/modules/authorized_key_module.html) module).